app/models/target.rb

```diff
@@ -10,14 +10,24 @@ class Target < ApplicationRecord
   validates :area_length, numericality: { greater_than: 0 }
   validate :user_must_have_less_than_maximum_targets
 
+  after_create :broadcast_to_compatible_users
+
   def compatible_targets
     Target.where(topic_id: topic_id)
           .where.not(user: user)
           .merge(Target.within(area_length, origin: self))
   end
 
+  def compatible_users
+    User.where id: compatible_targets.pluck(:user_id)
```

Comment:
Use parenthesis here. You can check here our criteria to know when to user parenthesis or not

https://github.com/rootstrap/tech-guides/tree/master/ruby#method-invocation-parens

Reply:
👌